### ISSUE TITLE
Replicate the experimental results

### ISSUE THREAD

**Issue by AiteAuburn** [lingxusb/Riboformer#6]
I was trying to replicate the experimental results based on the instructions provided in the README.md, but I’m getting a correlation near 0. Could you kindly let me know if there might be any step I missed? 

Here are the commands I ran:

conda env create -f env.yml
python data_processing.py
python training.py -e 15 -l 0.0005 --save

The following are my outputs:

Epoch 1/10
110/110 [==============================] - 9s 64ms/step - loss: 2.6865 - accuracy: 0.0026 - val_loss: 2.7077 - val_accuracy: 0.0000e+00
Epoch 2/10
110/110 [==============================] - 7s 62ms/step - loss: 2.3115 - accuracy: 0.0000e+00 - val_loss: 2.4763 - val_accuracy: 0.0000e+00
Epoch 3/10
110/110 [==============================] - 7s 62ms/step - loss: 2.1985 - accuracy: 0.0000e+00 - val_loss: 2.5019 - val_accuracy: 0.0000e+00
Epoch 4/10
110/110 [==============================] - 6s 53ms/step - loss: 2.1154 - accuracy: 0.0000e+00 - val_loss: 2.2920 - val_accuracy: 0.0000e+00
Epoch 5/10
110/110 [==============================] - 7s 63ms/step - loss: 2.0006 - accuracy: 0.0000e+00 - val_loss: 2.1798 - val_accuracy: 0.0000e+00
Epoch 6/10
110/110 [==============================] - 7s 63ms/step - loss: 1.8231 - accuracy: 0.0000e+00 - val_loss: 2.4897 - val_accuracy: 0.0000e+00
Epoch 7/10
110/110 [==============================] - 7s 63ms/step - loss: 1.6128 - accuracy: 0.0016 - val_loss: 2.4619 - val_accuracy: 0.0000e+00
Epoch 8/10
110/110 [==============================] - 7s 64ms/step - loss: 1.2935 - accuracy: 0.0034 - val_loss: 2.6008 - val_accuracy: 0.0000e+00
Epoch 9/10
110/110 [==============================] - 6s 55ms/step - loss: 1.0109 - accuracy: 0.0093 - val_loss: 2.7133 - val_accuracy: 0.0000e+00
Epoch 10/10
110/110 [==============================] - 7s 64ms/step - loss: 0.7921 - accuracy: 0.0156 - val_loss: 2.8072 - val_accuracy: 6.6667e-04
model prediction 0.002961881470921177
model training 0.9208136489518098
original correlation -0.003728394537975411

**Comment by lingxusb** [comment]
Hi, thank you for reaching out. I believe there are multiple training datasets in the data folder `/datasets/GSE119104_Mg_buffer/`
Please delete my test datasets/files which starts with `cm_mg_` and you will see this output:
```
--------------------------------------------------
Data loading
Current dictionary: /home/lingxusb_gmail_com/Riboformer/datasets/GSE119104_Mg_buffer/
205310 Training sequences
43996 Validation sequences
43995 Testing sequences
--------------------------------------------------
```

**Comment by AiteAuburn** [comment]
Thank you for the quick reply! After a quick check, it was indeed loading a different dataset earlier. So that is resolved! I’m currently trying to reproduce the results reported in Supplementary Table 1. However, after running the code multiple times, I'm consistently getting a correlation of around 0.77~0.78 for the GSE119104_Mg_buffer and 0.33 for the GSE139036_disome dataset. I just wanted to check if there are any data preparation details I might be missing to replicate the results.

**Comment by AiteAuburn** [comment]
Thank you for the quick reply! After a quick check, it was indeed loading a different dataset earlier.
